doc/release-notes-15620.md

```diff
@@ -0,0 +1,11 @@
+Updated RPCs
+------------
+
+* The `sendrawtransaction` and `testmempoolaccept` previouly accepted an
```

Comment:
In commit "doc: Add release notes for 15620" (faadaae):

This is a nice, detailed description, but I think it would be nice to have a sentence at the beginning summarizing the whole paragraph, like "The `-maxtxfee` setting no longer has any effect on non-wallet RPCs. [...]"

Reply:
Done